lua/sql.lua

```diff
@@ -410,25 +314,25 @@ function sql:update(...)
   if u.is_str(args[1]) then
     local tbl = args[1]
     local params = args[2]
-    assert_tbl(self, tbl)
+    assert_tbl(self, tbl, "update")
     unwrap_params(tbl, params)
   elseif u.is_str(args[1][1]) then
     local tbls = args[1]
     for k, tbl in ipairs(tbls) do
-      assert_tbl(self, tbl)
+      assert_tbl(self, tbl, "update")
       local params = args[k + 1]
       unwrap_params(tbl, params)
     end
   elseif args[1][2] == nil then
     local tbls = u.keys(args[1])
     for _, tbl in ipairs(tbls) do
-      assert_tbl(self, tbl)
+      assert_tbl(self, tbl, "update")
       local params = args[1][tbl]
       unwrap_params(tbl, params)
     end
   end
 
-  fail_on_wrong_input(ret_vals)
+  -- fail_on_wrong_input(ret_vals)
```

Comment:
Why? tho. Was a good catch. If we haven't parsed anything the input was wrong

Reply:
Yah I was thinking what if the user had a function to update table but the values either nil or no second argument is provided. So it will fail silently, or perhaps it would be better to ignore and return false if no params provided?